form/form_collections.rst

```diff
@@ -572,60 +555,57 @@ Now, you need to put some code into the ``removeTag()`` method of ``Task``::
     class Task
     {
         // ...
-
         public function removeTag(Tag $tag)
         {
             $this->tags->removeElement($tag);
         }
     }
 
-Template Modifications
-~~~~~~~~~~~~~~~~~~~~~~
-
 The ``allow_delete`` option means that if an item of a collection
 isn't sent on submission, the related data is removed from the collection
 on the server. In order for this to work in an HTML form, you must remove
 the DOM element for the collection item to be removed, before submitting
 the form.
 
-First, add a "delete this tag" link to each tag form:
+First, prepare a JavaScript function that adds a "delete this tag" link::
```

Comment:
oh, here we have  the same :)

Reply:
Maybe somewhere else too, cause I never understood the principle...